src/main/java/com/icthh/xm/ms/entity/service/XmEntitySpecService.java

```diff
@@ -99,6 +104,7 @@
     private final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
     private final ConcurrentHashMap<String, Map<String, TypeSpec>> typesByTenant = new ConcurrentHashMap<>();
     private final ConcurrentHashMap<String, Map<String, String>> typesByTenantByFile = new ConcurrentHashMap<>();
+    private final ConcurrentHashMap<String, Map<String, com.github.fge.jsonschema.main.JsonSchema>> dataSpecJsonSchemas = new ConcurrentHashMap<>();
```

Comment:
@sergeysenja1992 why do not import `com.github.fge.jsonschema.main.JsonSchema` class?

Reply:
now I see. too many imports here...